weekly/2023/week-46.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Week 46 in Packit
+date: 2023-11-20
+authors: lbarcziova
+tags:
+  - 2023-November
+  - 2023
+  - November
+---
+
+## Week 46 (November 14th – November 20th)
+
+- Packit now supports pre-release version in `propose_downstream` and `pull_from_upstream`.
+  A spec file update might be required, see the documentation for more details.
```

Comment:
is there something specific I can link here?

Reply:
Yes, after it's approved: https://github.com/packit/packit.dev/pull/777